src/simulation/m_riemann_solvers.fpp

```diff
@@ -1948,16 +1989,16 @@ contains
                                         end do
 
                                         flux_gsrc_rs${XYZ}$_vf(j, k, l, momxb + 1) = &
-                                            -xi_M*(rho_L*(vel_L(dir_idx(1))* &
-                                                          vel_L(dir_idx(1)) + &
-                                                          s_M*(xi_L*(dir_flg(dir_idx(1))*s_S + &
-                                                                     (1d0 - dir_flg(dir_idx(1)))* &
-                                                                     vel_L(dir_idx(1))) - vel_L(dir_idx(1))))) &
-                                            - xi_P*(rho_R*(vel_R(dir_idx(1))* &
-                                                           vel_R(dir_idx(1)) + &
-                                                           s_P*(xi_R*(dir_flg(dir_idx(1))*s_S + &
-                                                                      (1d0 - dir_flg(dir_idx(1)))* &
-                                                                      vel_R(dir_idx(1))) - vel_R(dir_idx(1)))))
+                                            -xi_M*(rho_L*(vel_L(idx1)* &
+                                                          vel_L(idx1) + &
```

Comment:
what motivated the change from dir_idx(1) to idx1?

Reply:
For consistency with `else` part. `else if (model_eqns == 2 .and. bubbles)` code is very similar to `else` code, so I wanted to make them have same things. I thought `idx1` is more concise than `dir_idx(1)`.

Reply:
There are more places using `dir_idx(1)` and `dir_idx(i)` in the code. If you agree, I will change them to `idx1` and `idxi`. Or, if you prefer `dir_idx(1)` and `dir_idx(i)`, I will change `idx1` and `idxi` to them. I just want to make them consistent.